core/src/i18n/en.yaml

```diff
@@ -550,7 +547,7 @@ common:
     hide: Hide Read-Only Fields
     k8s-name-input: "Name must consist of lowercase alphanumeric characters, can contain '-' (e.g.: 'my-name1')."
     key: Key must start and end with an alphanumeric character, can contain '-', '_' or '.'.
-    key-value: key=value, must start and end with an alphanumeric character, can contain '-', '_' or '.'.
+    key-value: key-value, must start and end with an alphanumeric character, can contain '-', '_' or '.'.
```

Comment:
I think it would be better to properly form it as "Both key and value must start", or something similar (also, it seems it's not entirely accurate - as KeyValueField is used both for labels and annotations... and while it's true for labels, it's not true for annotations... ugh)

Reply:
Ok, I will rephrase it :) Any suggestions how to make it more accurate?